Sim/SimG4Components/src/SimG4Alg.h

```diff
@@ -55,14 +55,14 @@ class G4SimAlg: public GaudiAlgorithm {
   private:
   DataHandle<fcc::MCParticleCollection> m_genParticles;
   /// Pointer to the interface of Geant simulation service
-  SmartIF<IG4SimSvc> m_geantSvc;
```

Comment:
The DataHandle is obsolete due to the event provider tool, apparently I forgot to delete that, could you remove it?